website/src/featureRequests/FeatureRequestsPlugin.js

```diff
@@ -15,10 +15,15 @@ function FeatureRequestsPlugin(context) {
   return {
     name: 'feature-requests-plugin',
     async contentLoaded({actions}) {
+      const basePath = normalizeUrl([context.baseUrl, '/feature-requests']);
+      await actions.createData('paths.json', JSON.stringify(basePath));
       actions.addRoute({
-        path: normalizeUrl([context.baseUrl, '/feature-requests']),
+        path: basePath,
         exact: false,
         component: '@site/src/featureRequests/FeatureRequestsPage',
+        modules: {
+          basePath: './feature-requests-plugin/default/paths.json',
```

Comment:
`createData` returns the module path, you don't need to hardcode it :)

Reply:
Haha, I never understood how these APIs work... I'll look deeper into how they're implemented and maybe improve the documentation on this😄